### PR TITLE
entending the documentation of extendedTree

### PR DESCRIPTION
**Pull request by craiglowe**
# Description
Additional documentation for the extended tree package.

## Relevant Links

- [Display Text](https://www.vertgenlab.org/)

### Checklist before requesting a review

- [ ] I performed a self-review of my code
- [ ] If it this a core feature, I have added thorough tests
- [ ] The command `go fmt` was used on all files included

### Testing

None

### Screenshots & Media

None

### Edge cases / Breaking Changes / Known Issues

None

### CODE REVIEW

File: expandedTree/expandedTree.go [hunk 1]
```diff
@@ -90,6 +94,7 @@ func GetBranch(node *ETree) []*ETree {
 	return branch
 }
 
+// GetBranch takes a root node and returns a slice of all leaf nodes in the tree.
```
Comment (rimangan) [inline expandedTree/expandedTree.go:97]: Copy paste error possibly

Comment (craiglowe) [reply expandedTree/expandedTree.go]: Thank you for catching this.  I just pushed a fix

**Review by cfauci — APPROVED** [review]

**Review by rimangan — APPROVED** [review]
